api/src/pcapi/connectors/typeform.py

```diff
@@ -143,7 +142,19 @@ def get_form(self, form_id: str) -> TypeformForm:
     def get_responses(
         self, form_id: str, num_results: int = 100, sort: str = "submitted_at,desc", since: datetime | None = None
     ) -> list[TypeformResponse]:
-        return []
+        return [
+            TypeformResponse(
+                response_id="",
+                date_submitted=datetime.utcnow(),
+                phone_number="0199000123",
+                email="responsemail@example.com",
+                answers=[
+                    TypeformAnswer(field_id="1a2b3c4d5-0000", choice_id=None, text="answer1"),
+                    TypeformAnswer(field_id="1a2b3c4d5-0001", choice_id=None, text="answer2"),
+                    TypeformAnswer(field_id="1a2b3c4d5-0002", choice_id=None, text="answer3"),
```

Comment:
Des questions sont définies dans `self.QUESTIONS`, c'est dommage que les réponses ne correspondent pas !
Je propose quelque chose ce cette forme, je te laisse choisir le texte :
```suggestion
                    TypeformAnswer(field_id=f"{form_id}-0000", choice_id=None, text="Patrick"),
                    TypeformAnswer(field_id=f"{form_id}-0001", choice_id=None, text="Dans la montagne"),
                    TypeformAnswer(field_id=f"{form_id}-0002", choice_id=None, text="J'en rêve depuis des années"),
```